src/desktop/apps/consign/client/analytics_middleware.ts

```diff
@@ -21,8 +21,9 @@ const analyticsMiddleware = store => next => action => {
       })
       return result
     }
+
     case actions.SUBMISSION_CREATED: {
-      analyticsHooks.trigger("consignment:submitted", {
+      analytics.track("consignment_submitted", {
```

Comment:
Should this be `consignment:submitted`?  Noticing that there's some idiom mixing between `:` and `_` here and below.

Reply:
The old version is a mediator trigger, but we are firing the event directly and use the event name.